### ISSUE TITLE
Show error when function not defined

### ISSUE THREAD

**Issue by kokizzu** [perl11/potion#79]
For example:

```
  123 println
```

will show nothing, that's unexpected..

**Comment by rurban** [comment]
Agreed. In historic context this was one of why's early decisions, to just return nil on missing methods.
I believe he wanted to allow catchable exceptions on those, but didn't have those implemented yet.

I'll see what I can do

**Comment by rurban** [comment]
Hmm, maybe add a strict mode to throw an error on call on nil.

**Comment by robotii** [comment]
We could just pretend everything is wrapped in an implicit Maybe monad?
